tests/unit/api/test_catalog.py

```diff
@@ -546,10 +546,10 @@ def test_get_catalog(catalog):
     # test list with include_id=True
     catalog_list = Catalog.list(include_id=True)
     expected_catalog_list["id"] = [
-        healthcare_catalog.id,
-        creditcard_catalog.id,
-        grocery_catalog.id,
-        default_catalog.id,
+        str(healthcare_catalog.id),
+        str(creditcard_catalog.id),
+        str(grocery_catalog.id),
+        str(default_catalog.id),
     ]
     assert_frame_equal(catalog_list, expected_catalog_list[catalog_list.columns])
```

Comment:
Actually line 549 to 552 can be remove as `id` field already appears in line 536 to 539.